esp_encrypted_img/idf_component.yml

```diff
@@ -1,4 +1,4 @@
-version: "1.0.2"
+version: "1.0.3"
```

Comment:
Though this PR does not introduce any breaking change, but still API is being modified here.

I think that we should bump up `major` (that is to make it `2.0.0`) version here.

This shall allow for time window to:

 - Modify example in ESP-IDF at https://github.com/espressif/esp-idf/tree/master/examples/system/ota/pre_encrypted_ota
 - Update its version dependency at https://github.com/espressif/esp-idf/blob/0440aca4e3c0608f13a6980c339e32587d78bf91/examples/system/ota/pre_encrypted_ota/main/idf_component.yml#L3 to `^2.0.0.`

(Until these changes, ESP-IDF will safely continue to use existing version of this component)

@MacDue What do you think?

Reply:
I think that's fine, I'll change that now :+1: